src/common/document-templates/statistiques-accompagnement-csv/statistiques-accompagnement-csv.spec.js

```diff
@@ -0,0 +1,256 @@
+const { buildExportStatistiquesCsvFileContent } = require('./statistiques-accompagnement-csv');
+
+describe('construction du contenu du fichier CSV d\'export des statistiques', () => {
+  it('devrait retourner le contenu à partir des données statistiques', () => {
```

Comment:
détail mais 'should' je le verrai mieux à l'impératif 'doit'

Reply:
Je pense que doit = must, le standard étant should devrait correspond mieux à should